dotnet/src/Connectors/Connectors.AI.HuggingFace/TextGeneration/HuggingFaceTextGenerationService.cs

```diff
@@ -29,6 +29,21 @@ public sealed class HuggingFaceTextGenerationService : ITextGenerationService
     private readonly string? _apiKey;
     private readonly Dictionary<string, object?> _attributes = new();
 
+    #region BreakingGlass Properties
+    /// <summary>
+    /// Model Id.
+    /// </summary>
+    public string? ModelId => this.GetModelId();
+
+    /// <summary>
+    /// Http Client used for the service call.
+    /// </summary>
+    /// <remarks>
+    /// The usage of this property is considered "unsafe". Use it only if strictly necessary.
+    /// </remarks>
+    public HttpClient Client => this._httpClient;
```

Comment:
Does this actually provide a benefit? It doesn't always have the endpoint built in, it doesn't add any headers that might be necessary. What can you do with this that you can't do with another HttpClient instance?

Reply:
Let's remove this to keep the public API surface minimal. We can add it back if needed.